2023-08-04 15:13:36,109 base_log.py[line:52] INFO test log
2023-08-04 15:13:36,109 base_log.py[line:53] ERROR failed
